Default to /cypress/logs/ directory for output

kristiehowboutdat opened the pull request:
Instead of creating log files in the project root directory, create them in `cypress/logs/`. Here are a few reasons why:

- easier to `gitignore`
- easier to upload as artifacts on CI alongside screenshots
- follows pattern of screenshots in Cypress that are created in the `cypress/screenshots/` folder

Partially addresses #18. It would be great to make this configurable, but I wasn't sure how you wanted to specify CLI args. This seemed like a sensible default for the time being.

bahmutov commented on src/index.js:
```diff
@@ -7,6 +7,7 @@ const reject = require('lodash.reject')
 const path = require('path')
 
 const cleanupFilename = s => kebabCase(deburr(s))
+const getFilepath = filename => `cypress/logs/${filename}`
```
could you use `path.join`, otherwise would not work on windows

bahmutov commented:
looks good to me, could you also update the README with this change?

kristiehowboutdat commented:
I used `path.join`, updated the `filepath` that is passed to the `failed-test` script so it has the full path instead of just the filename, and added the location to the README. Let me know if there are additional changes you'd like.

bahmutov commented:
Perfect, will merge now, hope NPM publish is working

bahmutov commented:
:tada: This PR is included in version 2.1.0 :tada:

The release is available on:
- `npm package (@latest dist-tag)`
- [GitHub release](https://github.com/bahmutov/cypress-failed-log/releases/tag/v2.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

souljacker commented:
Hey guys, any chance will we have the ability to config the logs path anytime soon?
